Add method for adding cb once on event in Data

redwind opened the pull request:
add once method on event in Data.js

redwind committed:
Add method for adding cb once on Data

add cb once on eventName

redwind committed:
remove Clear the collections on connected

charpeni commented on src/Meteor.js:
```diff
@@ -96,11 +96,11 @@ module.exports = {
-          Data.db[collection].remove({});
-        }
-      }
+      // if (Data.db && Data.db.collections) {
```
Doesn't look related to `once` method?

redwind replied:
@charpeni sorry, it's my wrong commit. let me revert that commit. 
Thanks !

redwind committed:
Revert "remove Clear the collections on connected"

This reverts commit 3dd8006de7f52ba626bd6161b978097aa2c77cef.

redwind committed:
Change import AsyncStorage

Use AsyncStorage from @react-native-community/async-storage instead from react-native

redwind committed:
update NetInfo

redwind committed:
update NetInfo

redwind committed:
migrate componentWillMount and componentWillUpdate

redwind committed:
Merge branch 'master' of https://github.com/redwind/react-native-meteor

merge update asyncStorage

redwind committed:
don't clear cache on connected for offline cache

redwind committed:
temporary add UNSAFE_ to support 0.6x
